base/src/org/compiere/model/POInfoColumn.java

```diff
@@ -64,9 +64,9 @@ public POInfoColumn (int ad_Column_ID, String columnName, String columnSQL, int
 		boolean isMandatory, boolean isUpdateable, String defaultLogic,
 		String columnLabel, String columnDescription,
 		boolean isKey, boolean isParent,
-		int ad_Reference_Value_ID, String validationCode,
+		int referencevalueId, String validationCode,
 		int fieldLength, String valueMin, String valueMax,
-		boolean isTranslated, boolean isEncrypted, boolean isAllowLogging, boolean isAllowCopy)
+		boolean isTranslated, boolean isEncrypted, boolean isAllowLogging, boolean isAllowCopy, String transactionName)
```

Comment:
This is a public constructor - should it be overloaded?  It is only called from the loadInfo method above to the risk is small.